__tests__/app.test.js

```diff
@@ -84,4 +84,39 @@ describe('get:', () => {
                 })
         });
     });
+    describe('/api/users', () => {
+        test('status:200 returns a single array', () => {
+            return request(app)
+                .get('/api/users')
+                .expect(200)
+                .then(res => {
+                    expect(res._body).toBeInstanceOf(Array);
+                });
+        });
+        test('status:200 returns a array of objects containing property: username, name & avatar_url', () => {
+            return request(app)
+                .get('/api/users')
+                .expect(200)
+                .then(res => {
+                    res._body.forEach(news => {
```

Comment:
suggest doing a length check before forEach to ensure you're not yielding a false positive